src/jobs/release-lerna-packages.yml

```diff
@@ -144,13 +150,17 @@ steps:
   # when dryrun is false: run post-job-steps
   - when:
       condition:
-        not: <<parameters.dryrun>>
+        and:
+          - equal: [ true, $NEEDS_RELEASE ]
```

Comment:
how does the env get persisted?

Reply:
@mshanemc  Comment needed to be fixed to reflect reality. Thank you.